tests/messages/freshmaker/build_state_change.json

```diff
@@ -0,0 +1,53 @@
+{
+    "body":{
+        "headers":{
+            "content-length": "756", 
+            "destination": "/topic/VirtualTopic.eng.freshmaker.build.state.changed", 
+            "JMS_AMQP_MESSAGE_FORMAT": "0", 
+            "JMS_AMQP_NATIVE": "false", 
+            "expires": "0", 
+            "priority": "4", 
+            "message-id": "ID:messaging-devops-broker01.web.prod.ext.phx2.redhat.com-42045-1527890187852-9:776487:0:0:1", 
```

Comment:
You should probably obscure this like you did in the other message for consistency